site/docs/master/aws-config.md

```diff
@@ -214,15 +221,17 @@ cat > velero-trust-policy.json <<EOF
     ]
 }
 EOF
-```
+    ```
 
 2. Create the IAM role:
-```bash
+
+    ```bash
 aws iam create-role --role-name velero --assume-role-policy-document file://./velero-trust-policy.json
```

Comment:
This doesn't render properly in GH, I believe the content needs to be indented to line up with the backticks.